WordPress/src/main/res/values/strings.xml

```diff
@@ -75,6 +75,11 @@
     <string name="upload">Upload</string>
     <string name="learn_more">Learn more</string>
     <string name="posting_post">Posting \"%s\"</string>
+    <string name="blog_posts">Blog Posts</string>
```

Comment:
I've updated the strings to have `my_site_header_` and `my_site_btn_` prefixes. I also realized that I have changed the `view_site` and `view_admin` strings in my previous commit which was a mistake. I think they are used elsewhere (we might remove them soon though), so I thought it's best to separate the strings. If we remove the UI elements that use those 2 values, we can get rid of them completely.